docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md

```diff
@@ -123,7 +123,21 @@ Follow these steps if you want to use your initial cluster to create and manage
    ```
 
    Refer to the initial config described earlier for the required and optional settings.
-   Ensure workload cluster object names (`Cluster`, `TinkerbellDatacenterConfig`, `TinkerbellMachineConfig`, etc.) are distinct from management cluster object names. Be sure to set the `managementCluster` field to identify the name of the management cluster. Keep the tinkerbellIP of workload cluster the same as tinkerbellIP of the management cluster.
+   Ensure workload cluster object names (`Cluster`, `TinkerbellDatacenterConfig`, `TinkerbellMachineConfig`, etc.) are distinct from management cluster object names. Keep the tinkerbellIP of workload cluster the same as tinkerbellIP of the management cluster.
+
+1. Be sure to set the `managementCluster` field to identify the name of the management cluster.
+
+   For example, the management cluster, _mgmt-demo_ is defined for our workload cluster _demo-workload-cluster_ as follows:
```

Comment:
nit: mind if we name mgmt and workload clusters to `_mgmt_` and `_w01_` just to stay consistent with the rest of the doc. Same for other providers

Reply:
Understood. Working on it

Reply:
Done. I derped up the commits a bit, but should be good now.